src/ui/public/vislib/lib/dispatch.js

```diff
@@ -231,6 +231,7 @@ define(function (require) {
       var label = this.getAttribute('data-label');
       if (!label) return;
       $('[data-label]', element.parentNode).not('[data-label="' + label + '"]').css('opacity', 0.5);
+      if (label === 'Count') $(this).css('opacity', 0.65);
```

Comment:
Would really rather not have === 'Count' here. This is going to make implementing custom legend labels irritating